pages/admin/request/create-replacement.tsx

```diff
@@ -58,19 +73,136 @@ export default function CreateReplacement() {
     billingPostalCode: '',
   });
 
+  const toast = useToast();
+  const router = useRouter();
+
+  const [getApplicant] = useLazyQuery<
+    GetApplicantReplacementResponse,
+    GetApplicantReplacementRequest
+  >(GET_APPLICANT_REPLACEMENT_QUERY, {
+    onCompleted: ({ applicant }) => {
+      setPermitHolderInformation({
+        firstName: applicant.firstName,
+        lastName: applicant.lastName,
+        email: applicant.email,
+        phone: applicant.phone,
+        addressLine1: applicant.addressLine1,
+        addressLine2: applicant.addressLine2,
+        city: applicant.city,
+        postalCode: applicant.postalCode,
+      });
+
+      const lastApplication = applicant.mostRecentApplication;
+      if (lastApplication !== undefined) {
```

Comment:
Code will fail here at runtime if `lastApplication` is `null`. The API you've written will return `null`, not `undefined` - in that case, this block will run and we will be accessing properties of `null`